source/platforms/snowflake.md

```diff
@@ -16,7 +11,121 @@ container Services](https://quickstarts.snowflake.com/guide/intro_to_snowpark_co
   (1 NVIDIA A10G - smallest NVIDIA GPU size available for Snowpark Containers to
   get started.)
 
-## Set up the Snowflake environment
+## `cuDF` and `cuML` in Snowflake Notebooks ML Runtime
+
+The [Snowflake Notebooks on Container Runtime for ML](https://docs.snowflake.com/en/developer-guide/snowflake-ml/notebooks-on-spcs)
+has `cuDF` and `cuML` built in in the environment.
```

Comment:
Should we mention here that if people want more control over their environment, or to install additional packages they should follow the Snowpark section instead?